src/common/integration_test_base_class.py

```diff
@@ -0,0 +1,79 @@
+# standard library
+import unittest
+
+# third party
+import mysql.connector
+
+# first party
+from delphi.epidata.client.delphi_epidata import Epidata
+from delphi.epidata.server._limiter import limiter
+
+
+class BasicIntegrationTest(unittest.TestCase):
+    """Basic integration test class"""
+
+    def __init__(self, methodName: str = "runTest") -> None:
+        super().__init__(methodName)
+        self.delete_from_tables_list = []
+        self.truncate_tables_list = []
+        self.create_tables_list = []
+        self.role_name = None
+        self.epidata = Epidata
```

Comment:
make this variable name a little more explanatory:
```suggestion
        self.epidata_client = Epidata
```